onfido-java/src/test/java/com/onfido/integration/WebhookEventVerifierTest.java

```diff
@@ -0,0 +1,55 @@
+package com.onfido.integration;
+
+import com.onfido.WebhookEventVerifier;
+import com.onfido.api.WebhookEvent;
+import com.onfido.api.WebhookObject;
+import com.onfido.exceptions.OnfidoException;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class WebhookEventVerifierTest {
+
+    String webhookToken = "_ABC123abc123ABC123abc123ABC123_";
+    WebhookEventVerifier verifier = new WebhookEventVerifier(webhookToken);
+
+    String rawEvent = "{\"payload\":{\"resource_type\":\"check\",\"action\":\"check.completed\",\"object\":{\"id\":\"check-123\",\"status\":\"complete\",\"completed_at_iso8601\":\"2020-01-01T00:00:00Z\",\"href\":\"https://api.onfido.com/v3/checks/check-123\"}}}";
+
+    WebhookEvent expectedEvent;
+
+    @BeforeMethod
+    public void setup() {
+        WebhookObject object = new WebhookObject();
+        object.id = "check-123";
```

Comment:
I think you should make all these properties `private final` and then add getters for consistency with the other models. I guess that means you'd have to do a load of individual `assertEquals` statements for each property